### PR TITLE
sensor readings endpoint with swagger documentation  added

### PR DESCRIPTION
**Pull request by amjadiftikhar**

### CODE REVIEW

File: src/server/api/controllers/sensor-readings.controller.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const knex = require('../../config/db');
+const Error = require('../lib/utils/http-error');
+
+const getSensorReadingByMaterialId = async (materialId) => {
+  try {
+    const sensorReading = await knex('sensor_readings')
```
Comment (cecastosic) [inline src/server/api/controllers/sensor-readings.controller.js:6]:
```suggestion
    const sensorReading = await knex('sensors_readings')
```

**Comment by cecastosic** [comment]
Great job @amjadiftikhar 💪

File: src/server/api/controllers/sensor-readings.controller.js (hunk 1 above)
Comment (amjadiftikhar) [reply src/server/api/controllers/sensor-readings.controller.js:6]: > 

I noticed the spelling mistake in table name, in today's class but you have already made a correction :) Thank you

**Comment by senner007** [comment]
@amjadiftikhar 
Can you resolve the merge conflict so we can get it merged

**Comment by cecastosic** [comment]
> @amjadiftikhar
> Can you resolve the merge conflict so we can get it merged

@senner007 The name of the table should also be changed to  sensors_readings (not sensor_readings)

**Comment by dpfernandes** [comment]
Solved on PR #150

**Pull request merged**